module/actor/witcherActor.js

```diff
@@ -493,6 +493,20 @@ export default class WitcherActor extends Actor {
         );
     }
 
+    async addItem(addItem, numberOfItem, forcecreate = false) {
+        let foundItem = this.items.find(item => item.name == addItem.name && item.type == addItem.type);
+        if (foundItem && !forcecreate && !foundItem.system.isStored) {
+            await foundItem.update({ 'system.quantity': Number(foundItem.system.quantity) + Number(numberOfItem) });
+        } else {
+            let newItem = { ...addItem.toObject() };
```

Comment:
@SquidOfDoomed the ``toObject`` here is the solution

Reply:
You mean to make it an object, then adjust it and then to call addItem?

Reply:
When making the copy, call toObject of the original